src/db/seeders/20210118210708-pet.js

```diff
@@ -0,0 +1,251 @@
+"use strict";
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.bulkInsert("Pets", [
+      {
+        name: "Morton",
+        age: 2,
+        weight: 14,
+        gender: "Male",
+        breed: "dog",
+        storeId: 1,
+        ownerId: 1,
+        hasShots: true,
+        createdAt: new Date(),
+        updatedAt: new Date(),
+      },
+      {
+        name: "Reynard",
+        age: 8,
+        weight: 22,
+        gender: "Male",
+        breed: "cat",
+        storeId: 3,
+        ownerId: null,
+        hasShots: true,
+        createdAt: new Date(),
+        updatedAt: new Date(),
+      },
+      {
+        name: "Sidoney",
+        age: 7,
+        weight: 39,
+        gender: "Female",
+        breed: "bird",
+        storeId: 1,
+        ownerId: 13,
+        hasShots: true,
+        createdAt: new Date(),
+        updatedAt: new Date(),
+      },
+      {
+        name: "Tyrone",
+        age: 2,
+        weight: 22,
+        gender: "Male",
+        breed: "snake",
+        storeId: 3,
+        ownerId: 15,
+        hasShots: true,
+        createdAt: new Date(),
+        updatedAt: new Date(),
+      },
+      {
+        name: "Grenville",
+        age: 2,
+        weight: 36,
+        gender: "Male",
+        breed: "dog",
+        storeId: 3,
+        ownerId: null,
+        hasShots: true,
+        createdAt: new Date(),
+        updatedAt: new Date(),
+      },
+      {
+        name: "Waverley",
+        age: 4,
+        weight: 20,
+        gender: "Male",
+        breed: "cat",
+        storeId: 1,
+        ownerId: 7,
+        hasShots: false,
+        createdAt: new Date(),
+        updatedAt: new Date(),
+      },
+      {
+        name: "Brendis",
+        age: 10,
+        weight: 40,
+        gender: "Male",
+        breed: "tutle",
+        storeId: 3,
+        ownerId: null,
+        hasShots: false,
+        createdAt: new Date(),
+        updatedAt: new Date(),
+      },
+      {
+        name: "Carling",
+        age: 1,
+        weight: 17,
+        gender: "Male",
+        breed: "bird",
+        storeId: 1,
+        ownerId: 12,
+        hasShots: true,
+        createdAt: new Date(),
+        updatedAt: new Date(),
+      },
+      {
+        name: "Ikey",
+        age: 9,
+        weight: 14,
+        gender: "Male",
+        breed: "dog",
+        storeId: 2,
+        ownerId: null,
+        hasShots: false,
+        createdAt: new Date(),
+        updatedAt: new Date(),
+      },
+      {
+        name: "Linell",
+        age: 3,
+        weight: 47,
+        gender: "Female",
+        breed: "cat",
+        storeId: 3,
+        ownerId: 9,
+        hasShots: false,
+        createdAt: new Date(),
```

Comment:
Its always good to keep track of what kind of date format your database is going to use and sticking with it. It won't really matter here for this project, but it's good to know. One option used is to store dates is UTC and then converting to the timezone on the frontend.